addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -143,6 +149,12 @@
             </DatiDDT>
         </DatiGenerali>
         <DatiBeniServizi>
+            <t t-if="record.move_type == 'in_refund' and document_type in ['TD16', 'TD17', 'TD18']">
```

Comment:
Can't we pass this value in from the python code?

Reply:
yep :+1: